### DISCUSSION TITLE
How to limit what gets installed ?

### DISCUSSION THREAD

**Discussion by mandarvaze** [abougouffa/minemacs#48] [Q&A] [answered]
I understand that `modules.el` is one way to prevent certain things from getting installed e.g. `me-email`
But `me-core-ui` installs modus, apropospriate and doom themes
If I'm only using doom-themes, is there a way to prevent installing modus and apropospriate ?

**Comment by abougouffa** [comment]
For the moment, there is no way to disable individual packages from modules!

**Comment by abougouffa** [comment, marked as answer]
@mandarvaze 

Starting from 4b91dc053c0d5f1b956aec1ee07a22f3c3c4d683, there is now a way to disable individual packages. This can be achieved by setting `minemacs-disabled-package` in the `modules.el` file. Any package added to this list will be ignored while loading MinEmacs' modules.
